vital/types/object_track_set.h

```diff
@@ -105,6 +112,16 @@ class VITAL_EXPORT object_track_state : public track_state
   {
     return time_;
   }
+  
+  ///Vector getters/setters for missing kw18 fields
+  vector_2d tracking_plane_loc() { return tracking_plane_loc_; }
+  vector_2d velocity() { return velocity_; }
+  vector_3d world_loc_xyz() { return world_loc_xyz_; }
+
+  void set_tracking_plane_loc( vector_2d tracking_plane_loc ) { tracking_plane_loc_ = tracking_plane_loc;}
+  void set_velocity( vector_2d velocity ) { velocity_ = velocity;}
+  void set_world_loc_xyz( vector_3d world_loc_xyz ) { world_loc_xyz_ = world_loc_xyz;}
+  ///////////////////////////////////////////////////////
 
```

Comment:
I don't think we use this style of ////////////////////// comments elsewhere in the code, I would remove them